client/layout/guided-tours/docs/ARCHITECTURE-FUTURE.md

```diff
@@ -0,0 +1,150 @@
+# Guided Tours: Architecture considerations for the future
+
```

Comment:
Add an explainer sentence here what this document is about and who would need or want to read it in what situation. Similar to your PR description should be fine.